client/verta/verta/endpoint/build/_build_hardware_compatibility.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+
+from verta._internal_utils import _utils
+
+
+class BuildHardwareCompatibilityNvidiaGPU:
+    def __init__(self, T4=None, V100=None, all=None):
+        self.T4 = T4
+        self.V100 = V100
+        self.all = all
```

Comment:
This class looks...strange to me. How is it being used? I can't seem to find it

Reply:
here: https://github.com/VertaAI/modeldb/pull/4003/files#diff-c02f63168a48821dc0b68a146b95ee0f8a2b1b631c1e8a8deca2b6ecfa9f4f41R58 it's literally just trying to match the spec here: dev.verta.ai/api/v1/deployment/swagger-ui/#/builds/get_builds__buildId_

Reply:
I see it's `return`ed by `BuildHardwareCompatibility.nvidia_gpu`, but where do we use `BuildHardwareCompatibility.nvidia_gpu` itself?